docker-compose/image/compute/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM neondatabase/compute-node:2221
```

Comment:
Oh, thanks. I'll change to use `Dockerfile.compute-node-v15`

Reply:
Just to be explicit: all `image: neondatabase/neon:2221` in docker-compose has to be pointed to the corresponding Docker files and their builds configured, otherwise we're still relying on some hardcoded tag that's not guaranteed to be stored for long.

Reply:
OK, thanks. I understood. I changed to use `compute-node-v14`  as default image.
But, I found my docker-compose doesn't work with `compute-node-v15` . I'll check the reason.
```
2022-10-24 10:04:38.477 GMT [39] FATAL:  database files are incompatible with server
2022-10-24 10:04:38.477 GMT [39] DETAIL:  The data directory was initialized by PostgreSQL version 14, which is not compatible with this version 15.0.
```

Reply:
fixed to specify `pg_version` when to create new timeline.